tests/unit/test_migration.py

```diff
@@ -98,11 +122,7 @@ def test_migration_loads_cloudfront_config(clean_db, cloudfront, fake_cf_client)
             ],
         },
     )
-    route = CdnRoute()
-    route.state = "provisioned"
-    route.domain_external = "example.gov"
-    route.dist_id = "sample-distribution-id"
-    migration = Migration(route, clean_db, fake_cf_client)
```

Comment:
There are a bunch of changes in this pattern. This is because the new service name fetching required a bit of mocking on every migration creation, so I created a fixture to make everything a bit more readable